core/controllers/classifier_test.py

```diff
@@ -52,22 +52,23 @@ def setUp(self):
         self.exploration = exp_services.get_exploration_by_id(self.exp_id)
 
         self.classifier_data = {
```

Comment:
Ditto, self.classifier_data_with_floats_stringified?

Actually I don't think this test looks right -- it has some float_values stuff. (So, technically right, but not what you want, probably...)

Reply:
Done